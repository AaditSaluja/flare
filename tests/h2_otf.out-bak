2019-09-18 17:46:26.491841
number of cpu cores: 4
cutoffs: [5]
kernel: two_body
number of hyperparameters: 3
hyperparameters: [1 1 1]
hyperparameter optimization algorithm: L-BFGS-B
uncertainty tolerance: 0.1 eV/A
timestep (ps): 0.0001
number of frames: 20
number of atoms: 2
system species: {'H'}
periodic cell: 
[[5. 0. 0.]
 [0. 5. 0.]
 [0. 0. 5.]]
previous positions (A):
H 2.30000000 2.50000000 2.50000000 
H 2.80000000 2.50000000 2.50000000 
--------------------------------------------------------------------------------

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 1 
wall time from start: 2.35 s 
