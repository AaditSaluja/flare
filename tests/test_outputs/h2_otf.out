2019-06-05 17:59:22.693863
number of cpu cores: 4
cutoffs: [5]
kernel: two_body
number of hyperparameters: 3
hyperparameters: [1 1 1]
hyperparameter optimization algorithm: L-BFGS-B
uncertainty tolerance: 0.1 eV/A
timestep (ps): 0.0001
number of frames: 20
number of atoms: 2
system species: {'H'}
periodic cell: 
[[5. 0. 0.]
 [0. 5. 0.]
 [0. 0. 5.]]
previous positions (A):
H 2.30000000 2.50000000 2.50000000 
H 2.80000000 2.50000000 2.50000000 
--------------------------------------------------------------------------------

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 1 
wall time from start: 0.61 s 

*-Frame: 0
Simulation Time: 0.000 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.30000000 2.50000000 2.50000000 	-22.29826208 0.00000000 0.00000000 	0.00000000e+00 0.00000000e+00 0.00000000e+00 	-1.07572762e+01 0.00000000e+00 0.00000000e+00 
H 2.80000000 2.50000000 2.50000000 	22.29826208 0.00000000 0.00000000 	0.00000000e+00 0.00000000e+00 0.00000000e+00 	1.07572762e+01 0.00000000e+00 0.00000000e+00 

temperature: 92.79 K 
kinetic energy: 0.011993 eV 
potential energy: 0.000000 eV 
total energy: 0.011993 eV 
wall time from start: 0.61 s 

Adding atom [0] to the training set.
Uncertainty: [0. 0. 0.].

GP hyperparameters: 
Hyp0 : Signal Std = 1.0030486268247016
Hyp1 : Length Scale = 0.9975360032150485
Hyp2 : Noise Std = 1e-06
likelihood: 21.269696772075278
likelihood gradient: [-9.93466919e-05  8.31870627e-05 -2.00000000e+06]
wall time from start: 3.24 s 
--------------------------------------------------------------------------------
-Frame: 1
Simulation Time: 0.000 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.29784854 2.50000000 2.50000000 	-22.25916591 0.00000000 0.00000000 	1.62348629e-01 0.00000000e+00 0.00000000e+00 	-3.22529676e+01 0.00000000e+00 0.00000000e+00 
H 2.80215146 2.50000000 2.50000000 	22.25916591 0.00000000 0.00000000 	1.62348629e-01 0.00000000e+00 0.00000000e+00 	3.22529676e+01 0.00000000e+00 0.00000000e+00 

temperature: 834.09 K 
kinetic energy: 0.107815 eV 
potential energy: 8.114377 eV 
total energy: 8.222192 eV 
wall time from start: 3.69 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 2 
wall time from start: 4.07 s 

*-Frame: 1
Simulation Time: 0.000 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.29784854 2.50000000 2.50000000 	-21.50874177 0.00000000 0.00000000 	1.62348629e-01 0.00000000e+00 0.00000000e+00 	-3.18909429e+01 0.00000000e+00 0.00000000e+00 
H 2.80215146 2.50000000 2.50000000 	21.50874177 0.00000000 0.00000000 	1.62348629e-01 0.00000000e+00 0.00000000e+00 	3.18909429e+01 0.00000000e+00 0.00000000e+00 

temperature: 815.47 K 
kinetic energy: 0.105408 eV 
potential energy: 8.114377 eV 
total energy: 8.219785 eV 
wall time from start: 4.07 s 

Adding atom [0] to the training set.
Uncertainty: [0.16234863 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.23466796413595045
Hyp1 : Length Scale = 0.21782341119121323
Hyp2 : Noise Std = 1e-06
likelihood: 45.952270014461746
likelihood gradient: [ 1.29142579e-04 -3.15310700e-04 -4.00000000e+06]
wall time from start: 4.09 s 
--------------------------------------------------------------------------------
-Frame: 2
Simulation Time: 0.000 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.29362181 2.50000000 2.50000000 	-19.89550114 0.00000000 0.00000000 	6.06100936e-02 0.00000000e+00 0.00000000e+00 	-5.18654537e+01 0.00000000e+00 0.00000000e+00 
H 2.80637819 2.50000000 2.50000000 	19.89550114 0.00000000 0.00000000 	6.06100936e-02 0.00000000e+00 0.00000000e+00 	5.18654537e+01 0.00000000e+00 0.00000000e+00 

temperature: 2156.91 K 
kinetic energy: 0.278801 eV 
potential energy: -2.508838 eV 
total energy: -2.230036 eV 
wall time from start: 4.09 s 
--------------------------------------------------------------------------------
-Frame: 3
Simulation Time: 0.000 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.28747545 2.50000000 2.50000000 	-17.42483708 0.00000000 0.00000000 	2.89105276e-01 0.00000000e+00 0.00000000e+00 	-6.98697800e+01 0.00000000e+00 0.00000000e+00 
H 2.81252455 2.50000000 2.50000000 	17.42483708 0.00000000 0.00000000 	2.89105276e-01 0.00000000e+00 0.00000000e+00 	6.98697800e+01 0.00000000e+00 0.00000000e+00 

temperature: 3914.29 K 
kinetic energy: 0.505961 eV 
potential energy: -2.738357 eV 
total energy: -2.232396 eV 
wall time from start: 4.09 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 3 
wall time from start: 4.45 s 

*-Frame: 3
Simulation Time: 0.000 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.28747545 2.50000000 2.50000000 	-18.03240537 0.00000000 0.00000000 	2.89105276e-01 0.00000000e+00 0.00000000e+00 	-7.01628872e+01 0.00000000e+00 0.00000000e+00 
H 2.81252455 2.50000000 2.50000000 	18.03240537 0.00000000 0.00000000 	2.89105276e-01 0.00000000e+00 0.00000000e+00 	7.01628872e+01 0.00000000e+00 0.00000000e+00 

temperature: 3947.20 K 
kinetic energy: 0.510215 eV 
potential energy: -2.738357 eV 
total energy: -2.228142 eV 
wall time from start: 4.45 s 

Adding atom [0] to the training set.
Uncertainty: [0.28910528 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.5024446853455423
Hyp1 : Length Scale = 0.2665016269049872
Hyp2 : Noise Std = 1e-06
likelihood: 70.48519587531096
likelihood gradient: [ 1.77792099e-05 -8.34039529e-05 -6.00000000e+06]
wall time from start: 4.47 s 
--------------------------------------------------------------------------------
-Frame: 4
Simulation Time: 0.000 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.27958923 2.50000000 2.50000000 	-15.78764295 0.00000000 0.00000000 	4.24239344e-02 0.00000000e+00 0.00000000e+00 	-8.64785803e+01 0.00000000e+00 0.00000000e+00 
H 2.82041077 2.50000000 2.50000000 	15.78764295 0.00000000 0.00000000 	4.24239344e-02 0.00000000e+00 0.00000000e+00 	8.64785803e+01 0.00000000e+00 0.00000000e+00 

temperature: 5996.42 K 
kinetic energy: 0.775097 eV 
potential energy: -2.367622 eV 
total energy: -1.592525 eV 
wall time from start: 4.47 s 
--------------------------------------------------------------------------------
-Frame: 5
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.27017974 2.50000000 2.50000000 	-13.58671933 0.00000000 0.00000000 	2.01771650e-01 0.00000000e+00 0.00000000e+00 	-1.00649556e+02 0.00000000e+00 0.00000000e+00 
H 2.82982026 2.50000000 2.50000000 	13.58671933 0.00000000 0.00000000 	2.01771650e-01 0.00000000e+00 0.00000000e+00 	1.00649556e+02 0.00000000e+00 0.00000000e+00 

temperature: 8122.66 K 
kinetic energy: 1.049935 eV 
potential energy: -2.643197 eV 
total energy: -1.593262 eV 
wall time from start: 4.47 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 4 
wall time from start: 4.84 s 

*-Frame: 5
Simulation Time: 0.001 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.27017974 2.50000000 2.50000000 	-13.26086180 0.00000000 0.00000000 	2.01771650e-01 0.00000000e+00 0.00000000e+00 	-1.00492354e+02 0.00000000e+00 0.00000000e+00 
H 2.82982026 2.50000000 2.50000000 	13.26086180 0.00000000 0.00000000 	2.01771650e-01 0.00000000e+00 0.00000000e+00 	1.00492354e+02 0.00000000e+00 0.00000000e+00 

temperature: 8097.31 K 
kinetic energy: 1.046658 eV 
potential energy: -2.643197 eV 
total energy: -1.596539 eV 
wall time from start: 4.84 s 

Adding atom [0] to the training set.
Uncertainty: [0.20177165 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.5917367976268267
Hyp1 : Length Scale = 0.2659225614669632
Hyp2 : Noise Std = 1e-06
likelihood: 95.78417524330341
likelihood gradient: [ 1.30685512e-05 -2.56020576e-06 -7.99999998e+06]
wall time from start: 4.87 s 
--------------------------------------------------------------------------------
-Frame: 6
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.25949076 2.50000000 2.50000000 	-10.75428734 0.00000000 0.00000000 	2.89086236e-02 0.00000000e+00 0.00000000e+00 	-1.12077902e+02 0.00000000e+00 0.00000000e+00 
H 2.84050924 2.50000000 2.50000000 	10.75428734 0.00000000 0.00000000 	2.89086236e-02 0.00000000e+00 0.00000000e+00 	1.12077902e+02 0.00000000e+00 0.00000000e+00 

temperature: 10071.98 K 
kinetic energy: 1.301903 eV 
potential energy: -8.955725 eV 
total energy: -7.653822 eV 
wall time from start: 4.87 s 
--------------------------------------------------------------------------------
-Frame: 7
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.24776416 2.50000000 2.50000000 	-8.18884371 0.00000000 0.00000000 	1.40068132e-01 0.00000000e+00 0.00000000e+00 	-1.21216573e+02 0.00000000e+00 0.00000000e+00 
H 2.85223584 2.50000000 2.50000000 	8.18884371 0.00000000 0.00000000 	1.40068132e-01 0.00000000e+00 0.00000000e+00 	1.21216573e+02 0.00000000e+00 0.00000000e+00 

temperature: 11781.45 K 
kinetic energy: 1.522870 eV 
potential energy: -9.177661 eV 
total energy: -7.654792 eV 
wall time from start: 4.87 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 5 
wall time from start: 5.23 s 

*-Frame: 7
Simulation Time: 0.001 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.24776416 2.50000000 2.50000000 	-8.55570260 0.00000000 0.00000000 	1.40068132e-01 0.00000000e+00 0.00000000e+00 	-1.21393556e+02 0.00000000e+00 0.00000000e+00 
H 2.85223584 2.50000000 2.50000000 	8.55570260 0.00000000 0.00000000 	1.40068132e-01 0.00000000e+00 0.00000000e+00 	1.21393556e+02 0.00000000e+00 0.00000000e+00 

temperature: 11815.88 K 
kinetic energy: 1.527320 eV 
potential energy: -9.177661 eV 
total energy: -7.650342 eV 
wall time from start: 5.24 s 

Adding atom [0] to the training set.
Uncertainty: [0.14006813 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.7360564991380012
Hyp1 : Length Scale = 0.25138706888181217
Hyp2 : Noise Std = 1e-06
likelihood: 120.20184768902651
likelihood gradient: [ 4.11047862e-04 -5.54817699e-03 -9.99999960e+06]
wall time from start: 5.29 s 
--------------------------------------------------------------------------------
-Frame: 8
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.23521205 2.50000000 2.50000000 	-6.57209155 0.00000000 0.00000000 	3.83532987e-02 0.00000000e+00 0.00000000e+00 	-1.28691607e+02 0.00000000e+00 0.00000000e+00 
H 2.86478795 2.50000000 2.50000000 	6.57209155 0.00000000 0.00000000 	3.83532987e-02 0.00000000e+00 0.00000000e+00 	1.28691607e+02 0.00000000e+00 0.00000000e+00 

temperature: 13279.30 K 
kinetic energy: 1.716482 eV 
potential energy: -5.031922 eV 
total energy: -3.315440 eV 
wall time from start: 5.29 s 
--------------------------------------------------------------------------------
-Frame: 9
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.22202584 2.50000000 2.50000000 	-5.03338362 0.00000000 0.00000000 	1.82497860e-01 0.00000000e+00 0.00000000e+00 	-1.34290398e+02 0.00000000e+00 0.00000000e+00 
H 2.87797416 2.50000000 2.50000000 	5.03338362 0.00000000 0.00000000 	1.82497860e-01 0.00000000e+00 0.00000000e+00 	1.34290398e+02 0.00000000e+00 0.00000000e+00 

temperature: 14459.88 K 
kinetic energy: 1.869083 eV 
potential energy: -5.183652 eV 
total energy: -3.314569 eV 
wall time from start: 5.29 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 6 
wall time from start: 5.66 s 

*-Frame: 9
Simulation Time: 0.001 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.22202584 2.50000000 2.50000000 	-4.65827897 0.00000000 0.00000000 	1.82497860e-01 0.00000000e+00 0.00000000e+00 	-1.34109437e+02 0.00000000e+00 0.00000000e+00 
H 2.87797416 2.50000000 2.50000000 	4.65827897 0.00000000 0.00000000 	1.82497860e-01 0.00000000e+00 0.00000000e+00 	1.34109437e+02 0.00000000e+00 0.00000000e+00 

temperature: 14420.93 K 
kinetic energy: 1.864049 eV 
potential energy: -5.183652 eV 
total energy: -3.319602 eV 
wall time from start: 5.66 s 

Adding atom [0] to the training set.
Uncertainty: [0.18249786 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.6626017348716184
Hyp1 : Length Scale = 0.2297601914445333
Hyp2 : Noise Std = 1e-06
likelihood: 145.06500672661008
likelihood gradient: [-2.67738353e-04 -1.62656084e-04 -1.19999981e+07]
wall time from start: 5.79 s 
--------------------------------------------------------------------------------
-Frame: 10
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.20839016 2.50000000 2.50000000 	-2.94881900 0.00000000 0.00000000 	4.29869606e-02 0.00000000e+00 0.00000000e+00 	-1.37779304e+02 0.00000000e+00 0.00000000e+00 
H 2.89160984 2.50000000 2.50000000 	2.94881900 0.00000000 0.00000000 	4.29869606e-02 0.00000000e+00 0.00000000e+00 	1.37779304e+02 0.00000000e+00 0.00000000e+00 

temperature: 15220.98 K 
kinetic energy: 1.967463 eV 
potential energy: -10.554024 eV 
total energy: -8.586561 eV 
wall time from start: 5.80 s 
--------------------------------------------------------------------------------
-Frame: 11
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.19446998 2.50000000 2.50000000 	-1.16511597 0.00000000 0.00000000 	2.05197452e-01 0.00000000e+00 0.00000000e+00 	-1.39763976e+02 0.00000000e+00 0.00000000e+00 
H 2.90553002 2.50000000 2.50000000 	1.16511597 0.00000000 0.00000000 	2.05197452e-01 0.00000000e+00 0.00000000e+00 	1.39763976e+02 0.00000000e+00 0.00000000e+00 

temperature: 15662.65 K 
kinetic energy: 2.024553 eV 
potential energy: -10.611707 eV 
total energy: -8.587154 eV 
wall time from start: 5.80 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 7 
wall time from start: 6.18 s 

*-Frame: 11
Simulation Time: 0.001 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.19446998 2.50000000 2.50000000 	-1.72789728 0.00000000 0.00000000 	2.05197452e-01 0.00000000e+00 0.00000000e+00 	-1.40035477e+02 0.00000000e+00 0.00000000e+00 
H 2.90553002 2.50000000 2.50000000 	1.72789728 0.00000000 0.00000000 	2.05197452e-01 0.00000000e+00 0.00000000e+00 	1.40035477e+02 0.00000000e+00 0.00000000e+00 

temperature: 15723.56 K 
kinetic energy: 2.032427 eV 
potential energy: -10.611707 eV 
total energy: -8.579280 eV 
wall time from start: 6.18 s 

Adding atom [0] to the training set.
Uncertainty: [0.20519745 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.6337327743880766
Hyp1 : Length Scale = 0.20953543823595455
Hyp2 : Noise Std = 1e-06
likelihood: 168.85610049238832
likelihood gradient: [-7.01560566e-06  2.03579897e-04 -1.39999932e+07]
wall time from start: 6.40 s 
--------------------------------------------------------------------------------
-Frame: 12
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.18038307 2.50000000 2.50000000 	-0.72739566 0.00000000 0.00000000 	6.69392387e-02 0.00000000e+00 0.00000000e+00 	-1.41219976e+02 0.00000000e+00 0.00000000e+00 
H 2.91961693 2.50000000 2.50000000 	0.72739566 0.00000000 0.00000000 	6.69392387e-02 0.00000000e+00 0.00000000e+00 	1.41219976e+02 0.00000000e+00 0.00000000e+00 

temperature: 15990.68 K 
kinetic energy: 2.066955 eV 
potential energy: -5.188800 eV 
total energy: -3.121845 eV 
wall time from start: 6.40 s 
--------------------------------------------------------------------------------
-Frame: 13
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.16622598 2.50000000 2.50000000 	-0.21880610 0.00000000 0.00000000 	3.11985410e-01 0.00000000e+00 0.00000000e+00 	-1.41676449e+02 0.00000000e+00 0.00000000e+00 
H 2.93377402 2.50000000 2.50000000 	0.21880610 0.00000000 0.00000000 	3.11985410e-01 0.00000000e+00 0.00000000e+00 	1.41676449e+02 0.00000000e+00 0.00000000e+00 

temperature: 16094.22 K 
kinetic energy: 2.080339 eV 
potential energy: -5.200807 eV 
total energy: -3.120468 eV 
wall time from start: 6.40 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 8 
wall time from start: 6.79 s 

*-Frame: 13
Simulation Time: 0.001 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.16622598 2.50000000 2.50000000 	0.35901949 0.00000000 0.00000000 	3.11985410e-01 0.00000000e+00 0.00000000e+00 	-1.41397690e+02 0.00000000e+00 0.00000000e+00 
H 2.93377402 2.50000000 2.50000000 	-0.35901949 0.00000000 0.00000000 	3.11985410e-01 0.00000000e+00 0.00000000e+00 	1.41397690e+02 0.00000000e+00 0.00000000e+00 

temperature: 16030.95 K 
kinetic energy: 2.072160 eV 
potential energy: -5.200807 eV 
total energy: -3.128647 eV 
wall time from start: 6.79 s 

Adding atom [0] to the training set.
Uncertainty: [0.31198541 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.5708642019656901
Hyp1 : Length Scale = 0.19853234968806485
Hyp2 : Noise Std = 1e-06
likelihood: 193.40452807717887
likelihood gradient: [ 1.16859429e-03 -5.56105195e-03 -1.59999846e+07]
wall time from start: 7.15 s 
--------------------------------------------------------------------------------
-Frame: 14
Simulation Time: 0.001 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.15210353 2.50000000 2.50000000 	1.31932336 0.00000000 0.00000000 	7.02636032e-02 0.00000000e+00 0.00000000e+00 	-1.40588013e+02 0.00000000e+00 0.00000000e+00 
H 2.94789647 2.50000000 2.50000000 	-1.31932336 0.00000000 0.00000000 	7.02636032e-02 0.00000000e+00 0.00000000e+00 	1.40588013e+02 0.00000000e+00 0.00000000e+00 

temperature: 15847.88 K 
kinetic energy: 2.048497 eV 
potential energy: -8.740557 eV 
total energy: -6.692061 eV 
wall time from start: 7.15 s 
--------------------------------------------------------------------------------
-Frame: 15
Simulation Time: 0.002 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.13810838 2.50000000 2.50000000 	2.53157011 0.00000000 0.00000000 	3.16907282e-01 0.00000000e+00 0.00000000e+00 	-1.38730239e+02 0.00000000e+00 0.00000000e+00 
H 2.96189162 2.50000000 2.50000000 	-2.53157011 0.00000000 0.00000000 	3.16907282e-01 0.00000000e+00 0.00000000e+00 	1.38730239e+02 0.00000000e+00 0.00000000e+00 

temperature: 15431.81 K 
kinetic energy: 1.994716 eV 
potential energy: -8.687642 eV 
total energy: -6.692926 eV 
wall time from start: 7.15 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 9 
wall time from start: 7.53 s 

*-Frame: 15
Simulation Time: 0.002 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.13810838 2.50000000 2.50000000 	1.80005661 0.00000000 0.00000000 	3.16907282e-01 0.00000000e+00 0.00000000e+00 	-1.39083141e+02 0.00000000e+00 0.00000000e+00 
H 2.96189162 2.50000000 2.50000000 	-1.80005661 0.00000000 0.00000000 	3.16907282e-01 0.00000000e+00 0.00000000e+00 	1.39083141e+02 0.00000000e+00 0.00000000e+00 

temperature: 15510.42 K 
kinetic energy: 2.004877 eV 
potential energy: -8.687642 eV 
total energy: -6.682765 eV 
wall time from start: 7.53 s 

Adding atom [0] to the training set.
Uncertainty: [0.31690728 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.5713344354187267
Hyp1 : Length Scale = 0.19049975609776812
Hyp2 : Noise Std = 1e-06
likelihood: 217.2402504825441
likelihood gradient: [-1.14057880e-01 -1.55524371e-03 -1.79999654e+07]
wall time from start: 7.69 s 
--------------------------------------------------------------------------------
-Frame: 16
Simulation Time: 0.002 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.12428690 2.50000000 2.50000000 	2.20276760 0.00000000 0.00000000 	6.31971039e-02 0.00000000e+00 0.00000000e+00 	-1.37152072e+02 0.00000000e+00 0.00000000e+00 
H 2.97571310 2.50000000 2.50000000 	-2.20276760 0.00000000 0.00000000 	6.31971039e-02 0.00000000e+00 0.00000000e+00 	1.37152072e+02 0.00000000e+00 0.00000000e+00 

temperature: 15082.71 K 
kinetic energy: 1.949591 eV 
potential energy: -4.841209 eV 
total energy: -2.891618 eV 
wall time from start: 7.69 s 
--------------------------------------------------------------------------------
-Frame: 17
Simulation Time: 0.002 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.11067796 2.50000000 2.50000000 	2.22454183 0.00000000 0.00000000 	3.52178285e-01 0.00000000e+00 0.00000000e+00 	-1.35016219e+02 0.00000000e+00 0.00000000e+00 
H 2.98932204 2.50000000 2.50000000 	-2.22454183 0.00000000 0.00000000 	3.52178285e-01 0.00000000e+00 0.00000000e+00 	1.35016219e+02 0.00000000e+00 0.00000000e+00 

temperature: 14616.61 K 
kinetic energy: 1.889342 eV 
potential energy: -4.779902 eV 
total energy: -2.890560 eV 
wall time from start: 7.70 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 10 
wall time from start: 8.07 s 

*-Frame: 17
Simulation Time: 0.002 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.11067796 2.50000000 2.50000000 	2.77362788 0.00000000 0.00000000 	3.52178285e-01 0.00000000e+00 0.00000000e+00 	-1.34751326e+02 0.00000000e+00 0.00000000e+00 
H 2.98932204 2.50000000 2.50000000 	-2.77362788 0.00000000 0.00000000 	3.52178285e-01 0.00000000e+00 0.00000000e+00 	1.34751326e+02 0.00000000e+00 0.00000000e+00 

temperature: 14559.31 K 
kinetic energy: 1.881936 eV 
potential energy: -4.779902 eV 
total energy: -2.897966 eV 
wall time from start: 8.07 s 

Adding atom [0] to the training set.
Uncertainty: [0.35217828 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.5714598957430113
Hyp1 : Length Scale = 0.18785774086347864
Hyp2 : Noise Std = 1e-06
likelihood: 242.00788577180154
likelihood gradient: [-2.79114710e-01  4.84701303e-01 -1.99999308e+07]
wall time from start: 8.40 s 
--------------------------------------------------------------------------------
-Frame: 18
Simulation Time: 0.002 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.09733664 2.50000000 2.50000000 	3.25667144 0.00000000 0.00000000 	5.12761361e-02 0.00000000e+00 0.00000000e+00 	-1.31842149e+02 0.00000000e+00 0.00000000e+00 
H 3.00266336 2.50000000 2.50000000 	-3.25667144 0.00000000 0.00000000 	5.12761361e-02 0.00000000e+00 0.00000000e+00 	1.31842149e+02 0.00000000e+00 0.00000000e+00 

temperature: 13937.45 K 
kinetic energy: 1.801554 eV 
potential energy: -7.728963 eV 
total energy: -5.927409 eV 
wall time from start: 8.41 s 
--------------------------------------------------------------------------------
-Frame: 19
Simulation Time: 0.002 ps 
El  Position (A) 				 GP Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.08430953 2.50000000 2.50000000 	3.95456852 0.00000000 0.00000000 	2.80201491e-01 0.00000000e+00 0.00000000e+00 	-1.28363254e+02 0.00000000e+00 0.00000000e+00 
H 3.01569047 2.50000000 2.50000000 	-3.95456852 0.00000000 0.00000000 	2.80201491e-01 0.00000000e+00 0.00000000e+00 	1.28363254e+02 0.00000000e+00 0.00000000e+00 

temperature: 13211.62 K 
kinetic energy: 1.707734 eV 
potential energy: -7.635760 eV 
total energy: -5.928026 eV 
wall time from start: 8.41 s 

Calling Quantum Espresso...
QE run complete.
number of DFT calls: 11 
wall time from start: 8.79 s 

*-Frame: 19
Simulation Time: 0.002 ps 
El  Position (A) 				 DFT Force (ev/A) 				 Std. Dev (ev/A) 					 Velocities (A/ps) 
H 2.08430953 2.50000000 2.50000000 	3.41217861 0.00000000 0.00000000 	2.80201491e-01 0.00000000e+00 0.00000000e+00 	-1.28624917e+02 0.00000000e+00 0.00000000e+00 
H 3.01569047 2.50000000 2.50000000 	-3.41217861 0.00000000 0.00000000 	2.80201491e-01 0.00000000e+00 0.00000000e+00 	1.28624917e+02 0.00000000e+00 0.00000000e+00 

temperature: 13265.54 K 
kinetic energy: 1.714703 eV 
potential energy: -7.635760 eV 
total energy: -5.921057 eV 
wall time from start: 8.79 s 

Adding atom [0] to the training set.
Uncertainty: [0.28020149 0.         0.        ].

GP hyperparameters: 
Hyp0 : Signal Std = 0.5716533954749471
Hyp1 : Length Scale = 0.18409778871137472
Hyp2 : Noise Std = 1e-06
likelihood: 266.42449275109504
likelihood gradient: [-7.89798748e-02  1.30234857e-03 -2.19998634e+07]
wall time from start: 9.35 s 
--------------------
Run complete. 
